Assets/Scripts/Model/Editions/XWAEdition.cs

```diff
@@ -8,5 +11,15 @@ public class XWAEdition : SecondEdition
         public override int MaxPoints { get { return 50; } }
 
         public XWAEdition() : base() { }
+
+        public override string GetPilotImageUrl(GenericShip ship, string filename)
```

Comment:
These filename parameters aren't used in the `SecondEdition` class either, does it make more sense to rip them out completely? I guess they're holdovers from 1.0 maybe?